examples/metadata-examples/sports-article-json-ld.amp.html

```diff
@@ -0,0 +1,153 @@
+<!doctype html>
+<!--
+      This sample AMP HTML file aims to be a minimalist document that
+      follows best practices and guidances for publishers to mark up
+      content for inclusion in various platforms.
+-->
+<html AMP lang="en">
+  <!-- you can use "amp" or "AMP" or "⚡" -->
+  <head>
+    <meta charset="utf-8">
+    <title>Lorem Ipsum</title>
+    <link rel="canonical" href="http://example.ampproject.org/article-metadata.html" />
+    <!--
+        The canonical document for this article should be linked, as above.
+
+        The canonical document should also have a corresponding <link> tag
+        within pointing at this AMP HTML file:
+
+          <link rel="amphtml" href="http://example.ampproject.org/article-metadata.amp.html" />
+
+        It is possible that this AMP HTML document is the canonical document
+        for this article, in which case, the canonical URL should point to this
+        document, and no "amphtml" link is required.
+    -->
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <!-- only one style tag is allowed, and it must have an "amp-custom" attribute -->
+    <style amp-custom>
+      body {
+        background-color: white;
+      }
+      amp-img {
+        background-color: gray;
+      }
+    </style>
+    <script type="application/ld+json">
+        //
+        // The document referenced in mainEntityOfPage should be the same as the
+        // canonical link above.
+        //
+        // Also, please be aware that some platforms that use AMP HTML have
+        // further restrictions with regards to some schema components.
+        //
+        // For example:
+        //
+        //   * The leader "image" referenced in the markup below must appear
+        //   somewhere on the AMP HTML document itself.
+        //
+        //   * The URL for that "image" must precisely match the src of the
+        //   amp-img tag.
+        //
+        //   * All marked-up URLs should be absolute.
+        //
+        //   * The "logo" dimensions must not exceed 600x60.
+        //
+      {
+        "@context": "http://schema.org",
+        "@type": "NewsArticle",
+        "mainEntityOfPage": "http://cdn.ampproject.org/article-metadata.html",
+        "headline": "Lorem Ipsum",
+        "url": "https://intense-heat-6570.firebaseapp.com/index.html",
+        "datePublished": "2016-04-01T12:00:00Z",
+        "dateModified": "2016-04-01T12:00:00Z",
+        "description": "The Vancouver Cunucks defeat the San Jose Sharks 4-2",
```

Comment:
Can you make sure the logos/text are all things we're able to use without license or attribution?  It might make sense to just use sample URLs and fictional names